index.js

```diff
@@ -1 +1,119 @@
 // implement your API here
+const express = require('express');
+const cors = require('cors');
+
+const users = require('./data/db.js');
+
+const server = express();
+
+server.use(express.json());
+server.use(cors());
+
+server.get('/', (req, res) => {
+  res.send('Get is working!!!');
+});
+
+server.get('/users', (req, res) => {
+  users
+    .find()
+    .then(users => {
+      res.status(200).json(users);
+    })
+    .catch(err => {
+      res
+        .status(500)
+        .json({ error: 'The users information could not be retrieved.' });
+    });
+});
+
+server.get('/users/:id', (req, res) => {
+  const { id } = req.params;
+
+  users
+    .findById(id)
+    .then(user => {
+      if (user) {
+        res.status(200).json(user);
+      } else {
+        res
+          .status(404)
+          .json({ message: 'The user with the specified ID does not exist.' });
+      }
+      res.status(200).json(users);
+    })
+    .catch(err => {
+      res
+        .status(500)
+        .json({ error: 'The user information could not be retrieved.' });
+    });
+});
+
+server.post('/users', (req, res) => {
+  const userInfo = req.body;
+  if (userInfo.name && userInfo.bio) {
```

Comment:
Noticed you did an 'if' statement with the '.then' inside, then below did the opposite. It's best to stick with one structure.